### PR TITLE
Fixed: Flash of 'No services found' on marketplace grid

### PR DESCRIPTION
**Pull request by sslotsky**
## Reason for change

The marketplace grid was flashing the "No services found" message instead of displaying the skeleton. This tightens up the logic and makes sure that the `filteredServices` function returns the skeleton services when they're provided.

## Testing

Test this in dashboard-render and make sure you see the skeleton when loading the marketplace, and that the "No services found" message does not flash at any point. Also make sure to check Storybook and the docs site.

### CODE REVIEW

File: src/components/manifold-marketplace-grid/manifold-marketplace-grid.tsx [hunk 1]
```diff
@@ -90,34 +89,17 @@ export class ManifoldMarketplaceGrid {
...
 
-    // Handle excludes
-    if (Array.isArray(this.excludes)) {
-      services = services.filter(
-        ({ body: { label } }) => this.excludes && !this.excludes.includes(label)
-      );
-    }
+    const services = this.products.length
+      ? this.services.filter(s => this.products.includes(s.body.label))
+      : this.services;
```
Comment (dangodev) [inline src/components/manifold-marketplace-grid/manifold-marketplace-grid.tsx]: Minor note: this reverses the ability to specify order of products, I think. Do we want to keep that?

File: src/components.d.ts [hunk 2]
```diff
@@ -331,16 +327,15 @@ export namespace Components {
     'templateLinkFormat'?: string;
   }
   interface ManifoldMarketplaceGrid {
-    'excludes'?: string[];
```
Comment (dangodev) [inline src/components.d.ts]: If this is being removed, docs should be updated

File: src/components/manifold-marketplace-grid/manifold-marketplace-grid.tsx (hunk 1 above)
Comment (sslotsky) [reply src/components/manifold-marketplace-grid/manifold-marketplace-grid.tsx]: Ah, I didn't think of that. Was that explicitly requested? I can restore it if it matters.

File: docs/docs/components/manifold-marketplace.md [hunk 3]
```diff
@@ -14,38 +14,23 @@ A list of all Manifold services.
...
+></manifold-marketplace>
 ```
 
 ## Hide categories
 
-Hiding the categories is a great way to have a more compact display. It’s
-recommended to use this if you’re only displaying a few products via
-`products` (above):
+Hiding the categories is a great way to have a more compact display. It’s recommended to use this if
+you’re only displaying a few products via `products` (above):
```
Comment (dangodev) [inline docs/docs/components/manifold-marketplace.md]: Why the changes here? Different prettier version?

Comment (sslotsky) [reply docs/docs/components/manifold-marketplace.md]: Probably the first time the file was saved since `proseWrap` was introduced.
